RunCat365/Program.cs

```diff
@@ -165,6 +167,18 @@ Func<T, bool> isChecked
             return new ToolStripMenuItem(title, null, [.. items]);
         }
 
+        private static Image? GetIconImageForMenu(Theme systemTheme, string title)
+        {
+            string prefix = (manualTheme == Theme.System ? systemTheme : manualTheme).GetString();
```

Comment:
Do we need to take into account the theme to determine the colors? Can't it always be black?